core/wiki/macros/list.tid

```diff
@@ -3,18 +3,18 @@ tags: $:/tags/Macro
 
 \define list-links(filter,type:"ul",subtype:"li",class:"",emptyMessage,field:"caption")
 \whitespace trim
-<$genesis $type=<<__type__>> class=<<__class__>>>
-<$list filter=<<__filter__>> emptyMessage=<<__emptyMessage__>>>
-<$genesis $type=<<__subtype__>>>
-<$link to={{!!title}}>
-<$let tv-wikilinks="no">
-<$transclude field=<<__field__>>>
-<$view field="title"/>
-</$transclude>
-</$let>
-</$link>
-</$genesis>
-</$list>
+<$genesis $type=<<__type__>> class=<<__class__>> >
```

Comment:
Why is there an extra space inserted before the closing `>`? We are striving for consistency and it's unhelpful to adopt a different convention.